cms/djangoapps/contentstore/tests/test_contentstore.py

```diff
@@ -114,6 +108,132 @@ def test_xlint_fails(self):
         err_cnt = perform_xlint(TEST_DATA_DIR, ['toy'])
         self.assertGreater(err_cnt, 0)
 
+    def test_assets_overwrite(self):
+        """ Tests that assets will similar 'displayname' will be overwritten during export """
```

Comment:
@adampalay  I am not sure, this what came out when this issue was raised. Can we get a thought from the TNL team?
As the test is passing, this means that assets with same `displayname` has been overwritten in the past and this is the first time we came aware of